ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/AbstractGetHeadersFromPeerTask.java

```diff
@@ -74,16 +74,19 @@ protected AbstractGetHeadersFromPeerTask(
     final List<BlockHeader> headers = headersMessage.getHeaders(protocolSchedule);
     if (headers.isEmpty()) {
       // Message contains no data - nothing to do
+      LOG.info("AbstractGetHeadersFromPeerTask: headers.isEmpty. Peer: {}", peer);
```

Comment:
I'm not sure why we are adding the class name in the class. the logger will add the name automatically in the log

Reply:
fixed!